lib/recurly/invoice.php

```diff
@@ -60,7 +60,18 @@ public function markSuccessful() {
   public function markFailed() {
     $this->_save(Recurly_Client::PUT, $this->uri() . '/mark_failed');
   }
-
+  
+  /**
+   * Enters an offline payment for an invoice
+   * @param array additional transaction attributes ('account', 'amount_in_cents', 'currency', 'description', 'accounting_code',  'payment_method', 'collected_at', 'tax_exempt', 'tax_code')
+   * @return Recurly_Transaction transaction on success
+   */
+  public function enterOfflinePayment($attributes = array(), $client = null) {
```

Comment:
It seems kind of awkward to pass in the attributes rather than a transaction object. What is the thinking here?